bin/late_afternoon_seal.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-`./bin/seal.rb tea quotes`
+./bin/seal.rb tea quotes
```

Comment:
Aha. I hit this one, too. Hence https://github.com/binaryberry/seal/pull/89/commits/b40f5d29b3371f31fe5f635b66f5862cf14c618c